aleph-client/src/pallets/staking.rs

```diff
@@ -424,13 +419,14 @@ impl<C: AsConnection + Sync> StakingRawApi for C {
     ) -> anyhow::Result<Vec<StorageKey>> {
         let key_addrs = api::storage().staking().eras_stakers_root();
         let mut key = key_addrs.to_root_bytes();
-        StorageMapKey::new(era, StorageHasher::Twox64Concat).to_bytes(&mut key);
-        self.as_connection()
-            .as_client()
-            .storage()
-            .fetch_keys(&key, 10, None, at)
-            .await
-            .map_err(|e| e.into())
+        key.extend(subxt::ext::sp_core::twox_64(&era.encode()));
+        key.extend(&era.encode());
```

Comment:
why do we extend the key with era twice?

Reply:
If I'm not mistaken, `EraIndex` and `AccountId` should be encoded using `twox64_concat`